### ISSUE TITLE
An incorrect address is shown in the document

### ISSUE THREAD

**Issue by vae520283995** [neo-project/docs#1334]
```
Format | Value
-- | --
Private Key | 087780053c374394a48d685aacf021804fa9fab19537d16194ee215e825942a0
Public Key (Compressed) | 03cdb067d930fd5adaa6c68545016044aaddec64ba39e548250eaea551172e535c
Script | 0c2103cdb067d930fd5adaa6c68545016044aaddec64ba39e548250eaea551172e535c4156e7b327
Address | NhZ5eahZAZ6UBsbCLcCQH6qqHdzuxt2HKa
```
https://github.com/neo-project/docs/blob/master/docs/en-us/basic/concept/wallets.md#ordinary-address
The correct address should be **NNLi44dJNXtDNSBkofB48aTVYtb1zZrNEs**

**Comment by vae520283995** [comment]
@chenzhitong

**Comment by Celia18305** [comment]
Fixed in #1344
